webapp/src/components/Vendor/decentraland/NFTTopbar/SelectedFilters/Pill/Pill.tsx

```diff
@@ -0,0 +1,15 @@
+import Icon from 'semantic-ui-react/dist/commonjs/elements/Icon'
```

Comment:
```suggestion
import { Icon } from 'decentraland-ui'
```
I believe we can use the one exported on our lib, it's just the semantic-ui one